test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresCapability.cs

```diff
@@ -1453,7 +1573,7 @@ public static int Method ()
 				}
 
 				// Bug https://github.com/dotnet/linker/issues/2379
-				[ExpectedWarning ("IL2046", ProducedBy = ProducedBy.Analyzer)]
+				//[ExpectedWarning ("IL2046", ProducedBy = ProducedBy.Analyzer)]
```

Comment:
If the warning is no longer expected, please delete this line and also the comment with the link to the bug (similarly above).